scss/civicrm/common/_forms.scss

```diff
@@ -328,3 +328,11 @@ input.error {
     border-color: $input-border-focus;
   }
 }
+
+// Add this class to style a form inside a container
+// like you see at /civicrm/civirule/form/rule?reset=1&action=add
+.crm-form-block-container {
+  .crm-form-block {
+    @include block-form();
```

Comment:
this looks like a global change, anything reported by backstop?

Reply:
`.crm-form-block-container` is not used anywhere **yet**, sorry, should have described a bit in more details. Basically, @tunbola will need this class to apply to his changes in a **custom** extension. I **cannot** do in Shoreditch smth like

```css
.tunbolas-custom-extension-wrapper {
  .crm-form-block {
    @include block-form();
```
because it is not related to vanilla civi. So I added a new class that he + other devs can use to style the form inside the accordion.